main.py

```diff
@@ -58,13 +59,21 @@ def search_book():
 
 
 def change_read():
-
     book_id = ui.get_book_id()
-    book = store.get_book_by_id(book_id)  
-    new_read = ui.get_read_value()     
-    book.read = new_read 
+    book = store.get_book_by_id(book_id)
+    new_read = ui.get_read_value()
+    book.read = new_read
     book.save()
-    
+
+
+def delete_book():
+    book_id = ui.get_book_id()
+    try:
+        book = store.get_book_by_id(book_id)
+        store._delete_book(book)
```

Comment:
It's unusual to call _ method directly. Note the comment here - do you see the other way you can delete a book? https://github.com/Baldeaglefacts/readinglist/blob/e7b462ef3986b08f26a62c290fb0ae277cc53ae3/bookstore.py#L82-L91